README.adoc

```diff
@@ -24,11 +24,12 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
 
 `odo` is designed to be simple and concise with the following key features:
 
-* Simple syntax and design centered around concepts familiar to developers, such as project, application, and component.
+* Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the OpenShift cluster for deployment.
-* Supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.
-* Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real-time.
-* Lists all available components and services from the OpenShift cluster.
+* Official full support of Node.js and Java components.
+* Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
```

Comment:
Run odo catalog list components to see -> Run `odo catalog list components` to see

Reply:
And remove "and unsupported components".

Reply:
Run odo catalog list components to see... needs to e removed from this conceptual module. It should be part of a procedural module.

Reply:
according to @kadel `odo catalog list components` can have different outputs depending on the cluster's configuration. I'd remove this command altogether from here and find a new place to put it